docs/source/tutorials/open_images_evaluation.ipynb

```diff
@@ -0,0 +1,444 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install Pillow tensorflow-hub"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Populate the following environment variables and run the inference script.\n",
+    "\n",
+    "This script is resumable and saves after every 10 samples are processed by default. It does\n",
+    "not process images in batches."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IMAGES_DIR=/PATH/TO/IMAGES\n",
+    "OUTPUT_DIR=/PATH/TO/PREDICTIONS\n",
+    "\n",
+    "MODEL_HANDLE=\"https://tfhub.dev/google/faster_rcnn/openimages_v4/inception_resnet_v2/1\"\n",
+    "# MODEL_HANDLE=\"https://tfhub.dev/google/openimages_v4/ssd/mobilenet_v2/1\"\n",
+    "\n",
+    "python open_images_eval/scripts/inference.py \\\n",
+    "    --output_dir ${OUTPUT_DIR} \\\n",
+    "    --output_format tf_object_detection_api \\\n",
+    "    ${IMAGES_DIR} ${MODEL_HANDLE}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the data into FiftyOne\n",
+    "\n",
+    "We are going to use the [fiftyone](https://github.com/voxel51/fiftyone) package\n",
+    "for visualizing the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Installing FiftyOne\n",
+    "\n",
+    "See [FiftyOne Installation](https://voxel51.com/docs/fiftyone/getting_started/install.html)\n",
+    "for up-to-date instructions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a persistent FiftyOne dataset\n",
+    "\n",
+    "The following script loads the data into a FiftyOne [Dataset](https://voxel51.com/docs/fiftyone/api/fiftyone.core.dataset.html#fiftyone.core.dataset.Dataset).\n",
+    "This process copies all labels and metadata to a non-relational database for rapid access and powerful querying, but only paths to the images are stored\n",
+    "in the database, not copies of the images themselves!\n",
+    "\n",
+    "The dataset is set to [persistent](https://voxel51.com/docs/fiftyone/user_guide/using_datasets.html#dataset-persistence)\n",
+    "so that it remains in the database and can be loaded in a new python process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET_NAME=\"open-images-v4-test\"\n",
+    "IMAGES_DIR=/PATH/TO/IMAGES\n",
+    "BOUNDING_BOXES_EXPANDED=/PATH/TO/test-annotations-bbox_expanded.csv\n",
+    "IMAGE_LABELS_EXPANDED=/PATH/TO/test-annotations-human-imagelabels-boxable_expanded.csv\n",
+    "PREDICTIONS_PATH=/PATH/TO/PREDICTIONS.csv\n",
+    "CLASS_DESCRIPTIONS=/PATH/TO/class-descriptions-boxable.csv\n",
+    "\n",
+    "python open_images_eval/scripts/load_data.py \\\n",
```

Comment:
Give the command for running this without predictions for people who just want to browse through the dataset.

Reply:
I wrote a comment in the code block

Reply:
You also need to remove `--load_images_with_preds`. Could you just make this an alternate code block to make it easier to copy and paste?